package.json

```diff
@@ -27,7 +27,9 @@
     "dist:modules": "modclean -r -n default:safe && yarn install --production",
     "predeploy": "npm-run-all build dist:*",
     "preversion": "auto-changelog  --config .auto-changelog.json --unreleased --commit-limit false --stdout --template preview.hbs",
-    "version": "auto-changelog -p --config .auto-changelog.json --unreleased && git add CHANGELOG.md"
+    "version": "npm-run-all bump:*",
+    "bump-openapi": "npm-run-all openapi:* && git add openapi/*.yaml",
```

Comment:
shouldn't this be
```suggestion
    "bump:openapi": "npm-run-all openapi:* && git add openapi/*.yaml",
```

Reply:
nice catch, thank you. done in https://github.com/pagopa/io-functions-lollipop/pull/13/commits/dbc64465aa4b1c56e6bbe5bee156325a287134f3